### PR TITLE
Display identicon when contributor has no avatar

### PR DESCRIPTION
**Pull request by CikiMomogi**
Before:
avatar where

After:
avatar fallback

### CODE REVIEW

**Review by roj1512 — COMMENTED** [review]
This is not bad, but identicons would look better, for example, `https://identicons.github.com/<username>.png`. What do you think?

**Comment by CikiMomogi** [comment]
That's a good idea. But it will make the client call the Github twice. The user might barely notice the difference, though. Are you okay with that?

**Comment by roj1512** [comment]
Yes, I am.

File: site/docs/public/icons/love.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<?xml version="1.0" encoding="UTF-8"?><svg id="Layer_2" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 107.61 107.63"><defs><style>.cls-1{fill:#fff;}.cls-2{fill:#f65066;}</style></defs><g id="Layer_2-2"><rect class="cls-1" x="17.37" y="20.67" width="71.81" height="67.97"/><path class="cls-2" d="m107.61,50.4c0,33.24-24.11,57.42-54.32,57.23-15.34-.1-28.47-6.12-38.91-17.41C4.03,79.03-.76,65.63.1,50.41,1.6,23.66,23.65-.33,54.69,0c15.28.16,28.4,6.05,38.8,17.36,9.4,10.22,13.95,22.46,14.12,33.04Zm-22.18-5.83c-.1-.96-.16-2.03-.34-3.08-.71-4.04-2.66-7.33-5.95-9.82-2.47-1.87-5.21-2.89-8.29-3.3-2.54-.34-4.96-.08-7.31.7-3.84,1.27-5.87,3.92-8.04,7.28-.89,1.37-1.6,1.35-2.37-.1-2.46-4.62-7.07-7.46-12.27-8.02-5.18-.56-9.77,1.15-13.49,4.71-4.39,4.2-6.15,9.41-5.3,15.53.59,4.22,2.2,7.98,4.42,11.52,3.29,5.24,7.57,9.63,12.35,13.46,4.03,3.23,8.36,6.08,12.61,9.03,1.26.88,2.7.86,4.02.05,4.52-2.76,8.92-5.68,13.02-9.07,4.37-3.61,8.29-7.62,11.39-12.38,3.25-4.99,5.23-10.41,5.56-16.51Z"/></g></svg>
```
Comment (KnorpelSenf) [inline site/docs/public/icons/love.svg:1]: Can this file be removed now?

Comment (CikiMomogi) [reply site/docs/public/icons/love.svg:1]: Yes

Comment (roj1512) [reply site/docs/public/icons/love.svg:1]: Do it then.

**Review by roj1512 — APPROVED** [review]
Thank you!

**Pull request merged**